src/app/static/src/app/apiService.ts

```diff
@@ -95,13 +76,13 @@ export class APIService<S extends string, E extends string> implements API<S, E>
     };
 
     private _handleAxiosResponse(promise: Promise<AxiosResponse>) {
-        return promise.then((response: AxiosResponse) => {
-            const success = response && response.data;
+        return promise.then((axioxResponse: AxiosResponse) => {
+            const success = axioxResponse && axioxResponse.data;
             const data = success.data;
             if (this._onSuccess) {
                 this._onSuccess(data);
             }
-            return data;
+            return axioxResponse.data;
```

Comment:
I can't remember which PR (it was quite recent) that I changed this to return only the data part of 
```{status: "success", errors: [], data: whatever}```
but I now couldn't see it being used anywhere. Anyway we now need to revert it back to returning the whole object so that we can pull out the version info.